its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -229,6 +229,10 @@ public static void runMSBuild(Orchestrator orch, Path projectDir, String... argu
     runMSBuild(orch, projectDir, Collections.emptyList(), TIMEOUT_LIMIT, arguments);
   }
 
+  public static void buildMSBuild(Orchestrator orchestrator, Path projectDir) {
+    runMSBuild(orchestrator, projectDir, Collections.emptyList(), TIMEOUT_LIMIT, "/t:Restore,Rebuild");
```

Comment:
Why not call this `runMSBuild` as well?
It looks like an overload to me.

Reply:
With `runMSBuild` you can execute any target or command. This method only restores and builds.

Overall I think we need a bigger rework of how the builds are done. I've extracted this method just to avoid code duplication.